docs/static/getting-started-with-logstash.asciidoc

```diff
@@ -105,6 +105,22 @@ it with:
 sudo apt-get update && sudo apt-get install logstash
 --------------------------------------------------
 
+Configure Logstash to automatically start during bootup. If your
+distribution is using SysV init, then you will need to run:
+
+[source,sh]
+--------------------------------------------------
+sudo update-rc.d logstash defaults 95 10
```

Comment:
`defaults` indicates the runlevel where `95 10` indicates where the script goes in startup and shutdown sequence